src/components/ErrorBanner/ErrorBanner.js

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+/**
+ * ErrorBanner component
+ *
+ * A component to display error messages
+ *
+ */
+
+/**
+ * @namespace fabric
+ */
+var fabric = fabric || {};
+/**
+ *
+ * @param {HTMLElement} container - the target container for an instance of ErrorBanner
+ * @constructor
+ */
+fabric.ErrorBanner = function(container) {
+    this.container = container;
+    this.init();
+};
+
+fabric.ErrorBanner.prototype = (function() {
+
+    var _clipper;
+    var _bufferSize;
+    var _textContainerMaxWidth = 700;
+    var _clientWidth;
+    var _textWidth;
+    var _initTextWidth;
+    var _chevronButton;
+    var _errorBanner;
+    var _actionButton;
+    var _closeButton;
+    var _bufferElementsWidth = 88;
+
+    /**
+     * sets styles on resize
+     */
+    var _onResize = function(event) {
+        _clientWidth = _errorBanner.offsetWidth;
+        if ((_clientWidth - _bufferSize) > _initTextWidth && _initTextWidth < _textContainerMaxWidth) {
+            _textWidth = "auto";
+            _chevronButton.style.display = "none";
```

Comment:
I'm wondering if we can just add a class to the chevronButton container to make it visible and have it display none by default instead of relying on js setting inline styles. Something like:

.ms-ErrorBanner-expand {
  display: none;

  &.is-visible {
    display: inline-block;
  }
}

and

var _onResize = function(event) {
        _clientWidth = _errorBanner.offsetWidth;
        if ((_clientWidth - _bufferSize) > _initTextWidth && _initTextWidth < _textContainerMaxWidth) {
            _textWidth = "auto";
            _chevronButton.className = "ms-ErrorBanner-expand";
            _collapse();
        } else {
            _textWidth = Math.min((_clientWidth - _bufferSize), _textContainerMaxWidth) + "px";
            _chevronButton.className += " is-visible";
        }
        _clipper.style.width = _textWidth;
    };